### PR TITLE
[FIX] Removed invalid usage of inherited member stdev2_ undefined in …

### PR DESCRIPTION
**Pull request by andreott**
…derived class GaussFitter1D.

### CODE REVIEW

**Comment by timosachsenberg** [comment]
nice catch. Do you know if this has any implications?

File: src/openms/source/TRANSFORMATIONS/FEATUREFINDER/GaussFitter1D.cpp [hunk 1]
```diff
@@ -75,37 +75,36 @@ namespace OpenMS
...
+    tmp.setValue("bounding_box:min", min_bb);
+    tmp.setValue("bounding_box:max", max_bb);
     tmp.setValue("statistics:mean", statistics_.mean());
     tmp.setValue("statistics:variance", statistics_.variance());
     model->setParameters(tmp);
 
     // fit offset
     QualityType quality;
-    quality = fitOffset_(model, set, stdev1_, stdev2_, interpolation_step_);
+    quality = fitOffset_(model, set, stdev, stdev, interpolation_step_);
```
Comment (cbielow) [inline src/openms/source/TRANSFORMATIONS/FEATUREFINDER/GaussFitter1D.cpp]: is this on purpose, i.e. a second bug?

**Comment by cbielow** [comment]
nice changes.
Can you say why none of the tests catched it!?

**Comment by andreott** [comment]
Yes, probably because the old test looks like this: 
START_SECTION((QualityType fit1d(const  RawDataArrayType &range, InterpolationModel *&model)))
	// dummy subtest
	TEST_EQUAL(1,1)
END_SECTION

I will add an appropriate test.

**Pull request merged**